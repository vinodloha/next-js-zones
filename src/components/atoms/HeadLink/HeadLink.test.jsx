import React from 'react'
import TestRenderer from 'react-test-renderer'
import HeadLink from '.'

describe('HeadLink', () => {
  it('creates a <link> element suitable for the document head', () => {
    const result = TestRenderer.create(
      <HeadLink
        rel="mockRel"
        type="mockType"
        sizes="mockSizes"
        href="http://example.com"
      />
    ).toJSON()

    expect(result).toMatchSnapshot()
  })
})
