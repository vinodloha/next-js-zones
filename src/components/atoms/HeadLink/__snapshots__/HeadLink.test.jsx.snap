// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeadLink creates a <link> element suitable for the document head 1`] = `
<link
  href="http://example.com"
  rel="mockRel"
  sizes="mockSizes"
  type="mockType"
/>
`;
